tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1179,6 +1203,24 @@ A base class static variable can also be accessed via a child class::
         B.x = 3
         prints(A.x, B.x) # 3 3
 
+.. warning::
+
+    Static variables declared on traits belong to the trait *itself*, and are *not*
+    inherited by classes that use the trait::
+
+        trait HasStatic:
+            static var static_var = 3
+
+        class UsingClass:
+            uses HasStatic
+
+        func _ready():
+            # Will work
+            print(HasStatic.static_var)
```

Comment:
As traits are implemented it will not. Consider traits as templates that when used by a class its variables, enum, functions, signals are copied over.